tests/test_accessibility.py

```diff
@@ -0,0 +1,26 @@
+"""
+Test accessibility auditing.
+
+You can run under phantomjs like this:
+SELENIUM_BROWSER=phantomjs ./run_tests.sh test_accessibility.py
+"""
+from bok_choy.web_app_test import WebAppTest
+from .pages import VisiblePage
+
+
+class AccessibilityTest(WebAppTest):
```

Comment:
Maybe also add a test for the browser exception (e.g., you're not using phantomjs). I figure that can be mocked for travis.